docs/tutorials/03_quantum_kernel.ipynb

```diff
@@ -62,19 +92,23 @@
       "text/plain": [
-       "<Figure size 360x360 with 1 Axes>"
+       "<Figure size 500x500 with 1 Axes>"
       ]
      },
-     "metadata": {
-      "needs_background": "light"
-     },
+     "metadata": {},
      "output_type": "display_data"
     }
    ],
    "source": [
-    "adhoc_dimension = 2\n",
+    "# Required imports\n",
+    "import matplotlib.pyplot as plt\n",
+    "import numpy as np\n",
+    "from qiskit_machine_learning.datasets import ad_hoc_data\n",
+    "\n",
+    "# Dataset: Ad hoc dataset for classification with 4 classes namely A train, A test, B train and B test.\n",
```

Comment:
I'd move out the comment from the cell to the description above. In general, if there's a comment in the code then it is good a candidate for the cell description. Same for all comments in the code.